### ISSUE TITLE
Adding .devcontainer and appending relevant commands to the Makefile

### ISSUE THREAD

**Issue by jahknem** [netbox-community/cookiecutter-netbox-plugin#3]
Recently I tried creating a plugin template similar to yours: https://github.com/jahknem/netbox-plugin-template 
Your project seems a lot more sophisticated. I think it could be useful to have an additional .devcontainer which could create a dev environment with a functioning netbox installation for testing and developing. To this end I would like to create a pull request which would include this function. Does this seem useful?

**Comment by arthanson** [comment]
@jahknem Yes, a PR would be great.  I was looking at adding that in the future and possibly a docker environment as well.  Please as it as an option in cookiecutter.json
